internal/reload.go

```diff
@@ -69,10 +75,29 @@ func Reload(ctx context.Context, doc *enigma.Doc, global *enigma.Global, silent
 }
 
 func logProgress(ctx context.Context, global *enigma.Global, reservedRequestID int, skipTransientLogs bool) {
+	InteractDef := enigma.InteractDef{
```

Comment:
Nit: You could do `InteractDef := &enigma.InteractDef{` so that it's already a pointer. It is more consistent with the overall style of the code.

Reply:
Fixed